features/user.feature

```diff
@@ -1,25 +1,8 @@
 Feature: Managing users
 
-  Background:
+  Scenario: I can add a new user
     Given I am logged in as an admin
     And I am on the Dashboard
-
-  Scenario: I can add a new user
-    Given I go to menu item "Users"
+    And I go to menu item "Users"
     When I click on the "Add New" link in the header
     Then I should be on the "Add New User" page
-
-    When I fill in "user_login" with "neweditor"
-    And I fill in "email" with "neweditor@example.org"
-    And I fill in "first_name" with "Ed"
-    And I fill in "last_name" with "New"
-    And I fill in "pass1" with "password"
-    And I fill in "pass2" with "password"
-    And I select "editor" from "role"
-    And I press "Add New User"
-    Then I should see a status message that says "New user created"
```

Comment:
It would be good to have a scenario which covers the "Then I should see a status message that says "New user created""  step - doesn't have to be this one.